src/test/java/org/opentripplanner/middleware/controllers/api/Auth0ConnectionTest.java

```diff
@@ -1,11 +1,12 @@
-package org.opentripplanner.middleware;
+package org.opentripplanner.middleware.controllers.api;
```

Comment:
I think this class should be in package `org.opentripplanner.middleware.auth` to match with `Auth0Connection`. The rationale is that tests performed in this file target `Auth0Connection#isCreatingSelf` and `Auth0Connection#isRequestingVerificationEmail` (despite the code in the tests dealing more with the api controllers than `Auth0Connection`).

Reply:
Moved in 695f54c4ffb08e23144a496d611b999c93c86764.